.github/workflows/test.yml

```diff
@@ -25,9 +25,9 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install pylint
+          pip install pylint pytest
       - name: Run unit tests
-        run: cd tests && python -m unittest -v
+        run: pytest
```

Comment:
FWIW, the previous `python -m unittest -v` will still run/find the tests so I'd be reluctant to introduce pytest as an additional external dependency unless there are other motivations (and/or we settle on pytest).

https://docs.python.org/3/library/unittest.html#unittest-test-discovery